textalyze.js

```diff
@@ -58,17 +58,16 @@ const main = () => {
   const args = process.argv.slice(2);
 
   args.forEach((arg) => {
-    fs.readFile(path.resolve(arg), (err, data) => {
+    fs.readFile(path.resolve(arg), 'utf8', (err, contents) => {
```

Comment:
Nice!  This is also what I did to make `readFile` give me a string rather than a `Buffer`.